lib/Doctrine/DBAL/Platforms/DrizzlePlatform.php

```diff
@@ -353,10 +356,10 @@ public function getListTableForeignKeysSQL($table, $database = null)
     /**
      * {@inheritDoc}
      */
-    public function getListTableIndexesSQL($table, $database = null)
+    public function getListTableIndexesSQL($table, $currentDatabase = null)
```

Comment:
Let's keep it as `$database` everywhere. The "current" aspect is irrelevant in the method signature.